libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialog.tsx

```diff
@@ -0,0 +1,27 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import { HubspotProvider } from "@aaronhayes/react-use-hubspot-form";
+
+import { Dialog } from "./Dialog";
+import {
+    ConversionTouchPointDialogBase,
+    IConversionTouchPointDialogBaseProps,
+} from "./ConversionTouchPointDialogBase";
+
+/**
+ * @public
+ */
+export const ConversionTouchPointDialog: React.FC<IConversionTouchPointDialogBaseProps> = (props) => {
```

Comment:
Should we rename this component `ConversionTouchPointDialog` → `HubspotConversionTouchPointDialog`, as the implementation of the component is tightly coupled with Hubspot?
※ I mean, to make it clearer for customers who don't use Hubspot that they shouldn't reuse this component.

Reply:
I have renamed them. Thanks!